src/views/search/chrome/AppHeader.tsx

```diff
@@ -27,6 +27,7 @@ export const AppHeader = () => {
                 href={appHeader.logo.link ?? "https://vectara.com/"}
                 target="_blank"
                 className="appHeaderLogo"
+                rel="noreferrer"
```

Comment:
I think we want to be able to track links between our demos and our marketing site. Why add this?

Reply:
Without this line, eslint outputs this warning and the fix was to add this line. Up to you whether you want to keep it or not.

```
Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations
```

I'll make sure to add some more details about additions to the code next time.

Reply:
Got it. Let's remove it.

Reply:
Removed!